docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,350 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+*******************************************************
+Get a List of Course Blocks in a Specified Course
+*******************************************************
+
+The endpoint to get a list of course blocks in a specified block tree
```

Comment:
Is a block tree always a course? Maybe just "list of course blocks in a course"?